docs/pages/build/submitting-to-app-stores.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Submitting builds to App Stores
+---
+
+This guide outlines how to submit your app to Apple App Store or Google Play Store. A paid developer account is required for each platform for which you wishy to submit an app. You can create an Apple Developer account on [Apple's developer site](https://developer.apple.com/account/) and a Google Play Developer account on the [Google Play Console sign-up page](https://play.google.com/apps/publish/signup/).
+
+## 1. Build a standalone app
+
+You'll need a built native app binary. You can either use [EAS Build](introduction.md) service or do it on your own.
+
+## 2. Start the submission
+
+To submit the previously built app to the appropriate app store, you simply run `eas submit -p ios` or `eas submit -p android` from inside of your project directory. Except specifying platform, no other flags are required, the command will lead you step by step. To see all possible options along with example usages, run `eas submit --help`.
+
+> Although it's possible to upload any binary to the store, each submission is assigned to an Expo project. That's why it's important to start submission from inside your project's directory - that's where your [app configuration](../workflow/configuration.md) is defined.
+
+### 2.1 Submitting your Android app to Google Play
+
+**Important:**
+
+- You have to create a [Google Service Account](https://cloud.google.com/iam/docs/creating-managing-service-accounts) and download its JSON private key.
+- After that, you'll have to create an app on [Google Play Console](https://play.google.com/apps/publish/) and upload your app manually at least once.
+
+#### Creating a Google Service Account
+
+See [expo.fyi/creating-google-service-account](https://expo.fyi/creating-google-service-account) to learn more.
+
+#### Manually uploading your app for the first time
+
+Before using `eas submit -p android` for uploading your standalone app builds, you have to upload your app manually at least once.<br />
+See [expo.fyi/first-android-submission](https://expo.fyi/first-android-submission) to learn more.
+
+#### Using EAS CLI to upload the further builds of your app
+
+To upload your iOS app to App store, run `eas submit --platform android` and follow the instructions on the screen. The command will do the following steps respectively:
+
+1. Log in to EAS account and ensure that your app project exists on EAS servers
+2. Prompt for Android package name unless `expo.android.package` in app configuration or `--android-package` param is provided.
+3. Ask for binary to submit. You can select one of the following:
+
+   - The latest finished iOS build for the project on EAS servers
+   - Specific build ID. The ID can be found on your EAS Account under _Builds v2_ section
```

Comment:
outside of the scope of this PR, but this reminds me that we should go ahead and rename this to EAS Build

Reply:
I assumed that it's going to be called _EAS Builds_ and already renamed it in the doc.